libs/server/Resp/AdminCommands.cs

```diff
@@ -363,24 +340,26 @@ private bool ProcessAdminCommands<TGarnetApi>(ReadOnlySpan<byte> command, ReadOn
                         SendAndReset();
                 }
             }
-            else if (command.SequenceEqual(CmdStrings.RESET))
-            { }
-            else if (command.SequenceEqual(CmdStrings.QUIT))
+            else if (command == RespCommand.RESET)
+            {
+                // XXX: Not Implemented?
```

Comment:
Shouldn't we return something here?

Reply:
I have fully removed the command for now. When we decide to implement the command, we can add it back to the parsing loop.